<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QWidget" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="DrawWidget" name="widget_CurInput" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_CurInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_SampleImg" native="true">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>100</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_SampleImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_ParmsSet">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>910</y>
     <width>800</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_ExplosureTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光时间（us）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FpsSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>帧数（fps）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ExplosureTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FpsSet">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ParamsSet">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>设置参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ShiftToMainWindow">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>切换到主界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestGetSingleImg_Take">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>采单帧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OutputBaseLine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>输出基线信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReadBaseLineInfo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>读取基线信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_CamOperate">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>910</y>
     <width>800</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_OpenCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartGrab">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopGrab">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>停止采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CloseCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ListDevice">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>枚举设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GetImg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>单帧图片采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CalibCam">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>相机一键标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PlaneFit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>激光平面标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReadCalibParms">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>读取标定参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearCalibParms">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>清空标定参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeasureSingleF">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>单帧测量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeasureContinuous">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>连续测量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SetBaseLine">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>设定基准线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ShowCurParams">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>显示当前参数</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_WinInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>1700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_NumOfCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>已连接相机数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CamIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>当前相机序号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumOfCam">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CamIndex">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_FilePath">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文件存储位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FilePath">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_FileName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文件保存名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FileName">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CalibrateData">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>标定文件目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibrateData">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BoardWidth">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>标定板宽角点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BoardWidth">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BoardHeight">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>标定板高角点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BoardHeight">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_SquareSize">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>标定方格大小</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SquareSIze">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CalibrateDataNum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>标定文件数量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibrateDataNum">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CalibLaserPlaneData">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>激光标定文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CalibLaserPlaneData">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header location="global">drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
